build-system/eslint-rules/reject-with-error.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var astUtils = require('eslint/lib/ast-utils');
+
+function leadingCommentMatches(leadingComments, regex) {
+  if (!leadingComments) {
+    return false;
+  }
+
+  return leadingComments.some(function(comment) {
+    return regex.test(comment.value);
+  });
+}
+
+function isLogCreateErrorCall(node) {
+  if (node.type != 'CallExpression') {
+    return false;
+  }
+
+  var call = node.callee;
+  return call.type == 'MemberExpression' &&
```

Comment:
not sure where my comments went. (think i commented last week)

can you update to either double or triple for consistency.